geoapps/utils/write_default_uijson.py

```diff
@@ -163,9 +163,17 @@ def write_default_uijson(path: str | Path, use_initializers=False):
     tipper_init = tipper_init if use_initializers else {}
 
     from geoapps.octree_creation.constants import app_initializer as oct_init
-
-    oct_init["geoh5"] = str(assets_path() / "FlinFlon.geoh5")
-    oct_init = oct_init if use_initializers else {}
+    from geoapps.octree_creation.constants import template_dict
```

Comment:
How did this get in here?  I'm not opposed to adding three levels by default, but I think we had a single refinement by default to keep it consistent with Kris's ui.jsons.

Reply:
Yea maybe, I will push back on Kris to get it full. I find it very annoying to have to modify the file every time. Don't think he will care since he has his own builder now...

Reply:
ok